source/shared/cpp/ObjectModel/Enums.h

```diff
@@ -371,7 +371,8 @@ namespace AdaptiveSharedNamespace
         RequiredPropertyMissing,
         InvalidPropertyValue,
         UnsupportedParserOverride,
-        IdCollision
+        IdCollision,
+        CustomError,
```

Comment:
do enum changes require a re-swig?

Reply:
Indeed.

---
In reply to: [316819721](https://github.com/microsoft/AdaptiveCards/pull/3375#discussion_r316819721) [](ancestors = 316819721)